ansible/provision.yaml

```diff
@@ -26,18 +26,37 @@
   pre_tasks:
     - include_vars: vars/flatcar/flatcar.yaml
       when: ansible_os_family == "Flatcar"
-  roles:
-    - role: offline
-    - role: packages
-    - role: fips
-    - role: kubeadm
-    - role: version
-    - role: gpu
-    - role: config
-    - role: networking
-    - role: providers
-    - role: images
-      when: download_images
-    - role: sysprep
-      when: sysprep # make the finalize process skipable
+  tasks:
+  - block:
+      - import_role:
+          name: offline
+      - import_role:
+          name: packages
+      - import_role:
+          name: fips
+      - import_role:
+          name: kubeadm
+      - import_role:
+          name: version
+      - import_role:
+          name: gpu
+      - import_role:
+          name: config
+      - import_role:
+          name: networking
+      - import_role:
+          name: providers
+      - import_role:
+          name: images
+        when: download_images
+      - import_role:
+          name: sysprep
+        when: sysprep # make the finalize process skipable
+    rescue:
+      - name: RHEL unsubscribe
+        redhat_subscription:
+          state: absent
+        when:
+          - ansible_distribution == 'RedHat'
```

Comment:
Let's only unsubscribe if KIB subscribed. Please use similar tasks to https://github.com/mesosphere/konvoy-image-builder/blob/592cefaa130c8b27ce85bf9e4d6f51205e10a690/ansible/roles/sysprep/tasks/redhat.yml#L86-L102